src/OpenDebugAD7/TextPositionTuple.cs

```diff
@@ -41,6 +41,7 @@ public static TextPositionTuple GetTextPositionOfFrame(PathConverter converter,
                     Path = convertedFilePath,
                     Name = Path.GetFileName(convertedFilePath)
                 };
+
```

Comment:
nit: please remove this file from the review since it is just a whitespace change.

Reply:
Ah a remnant from previous changes.